.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ['3.2', '3.1', '3.0', '2.7', '2.6']
+        ruby-version: ['3.4', '3.3', '3.2', '3.1', '3.0', '2.7']
```

Comment:
I ended up making a separate PR for this stuff: https://github.com/simonc/memfs/pull/49